### ISSUE TITLE
Improve 3D model of the top of the nose on the left side

### ISSUE THREAD

**Issue by legoboyvdlp** [c172p-team/c172p-detailed#1027]
The 3D model of the top of the nose has a strange shape on the left side and is a bit buggy, see: 

![image](https://user-images.githubusercontent.com/10434911/37911424-ef21018a-3107-11e8-8879-82e0ec14d3f4.png)

**Comment by gilbertohasnofb** [comment]
The rudder is not mistmatching in my opinion, you are comparing a deflected rudder with the back tip of the fuselage. They will never match like that.

The bottom image is a known issue though, but there isn't much I can do via textures, that's a 3D model problem which I thought of letting go. It affects only a single livery and is quite minor (though a tiny bit annoying). If @wlbragg agrees that this is not worth tackling, then I would suggest closing this as a `won't fix`, else I would be happy to test a PR with a mesh fix.

**Comment by legoboyvdlp** [comment]
Right :)

**Comment by gilbertohasnofb** [comment]
Confirmed, the matching in the rudder is actually pretty good:

![fg](https://user-images.githubusercontent.com/5700795/37933675-cfb32e9a-3143-11e8-8d83-42fafcfaeab8.jpg)

@wlbragg it's up to you if you want to take a look at that place on the left side of the nose or not. If not, do close this as won't fix.

**Comment by wlbragg** [comment]
>@wlbragg it's up to you if you want to take a look at that place on the left side of the nose or not. If not, do close this as won't fix.

I'll take a look and report back.

**Comment by wlbragg** [comment]
I was able to clean it up. I'll push it with some of the other corrections here in a bit.

**Comment by gilbertohasnofb** [comment]
@wlbragg Great, I will check that later. Many, many thanks!

**Comment by gilbertohasnofb** [comment]
I renamed and rephrased this issue for future reference, as well as removed the references to the rudder.

**Comment by gilbertohasnofb** [comment]
@legoboyvdlp I just noticed some yellow spots inside the air intake in your screenshot, what is that? Is it from some hotspot or is it some other bug I did not realise so far?

**Comment by legoboyvdlp** [comment]
That's the hotspots appearing through the cowling, as far as I know.

**Comment by gilbertohasnofb** [comment]
@legoboyvdlp Great, thanks for the info. So in that case the current PR will solve everything concerning this issue :+1:

**Comment by legoboyvdlp** [comment]
ZS-MDF:
![image](https://user-images.githubusercontent.com/10434911/37984086-310d8ccc-31ed-11e8-9519-0d5fd51c2ab2.png)

**Comment by legoboyvdlp** [comment]
![image](https://user-images.githubusercontent.com/10434911/37984105-43d9686c-31ed-11e8-96ae-a607464d548f.png)

**Comment by gilbertohasnofb** [comment]
@legoboyvdlp Thanks for the second image, that's really appreciated! That said, there is little we can do to improve this issue: the top bit of the nose is a separate object and it goes all the way to the bottom part of the side of the cockpit glass (where you marked in red on the left in the image above). The matching of the 'shading' layer is not matching there in GIMP, and though I tried to improve it it's just never going to be perfect. Normally a 3D model is texturized after the UV-map is completed and only minor tweaks might be left, but in our case we inherited a completely broken 3D model. So it will be really tough to improve this one I think... Concerning the line on the right side, what is the problem with it exactly? If it's because it is a bit fuzzy and not too straight, that's again the limitation of what can be achieved by textures. I've spent countless hours trying to get that looking to the best of my ability.
